### ISSUE TITLE
Potential issue with calculating total points in a year

### ISSUE THREAD

**Issue by KORraNpl** [ctftime/ctftime.org#121]
Hey guys,
I believe there might be an issue with summing up points for teams. I checked two methods as I wasn't able to find exact method ctftime is using:
* taking points from top10 CTFs of the team and adding points for all organized events on top of that
* putting organized CTFs on list together with rest of events and then calculating points for top10.

Neither method produced points visible at CTFtime. For example, using first method:
1. Dragon Sector 1081,284
1. Balsn 1053,526
1. PPP 943,715

Could you clarify this?

**Comment by theKidOfArcrania** [comment]
Yes you would compute organized CTFs as a list together with rest of events (2nd approach), but there are two pitfalls noted here

1. The weight displayed under "Organized events" is actually the weight voted this year, not the actual weight in the current year. This has been mentioned before but never resolved, see issue #51 
2. Organizers get the _same amount of points_ as first place would get on that same event (basically double the base weight). This generally holds for all events

Dragon sector would have the following events:
```
    Dragon CTF 2019                             198.000
  1 0CTF/TCTF 2019 Quals                        159.500
  2 Google Capture The Flag 2019 (Quals)        145.125
  2 ASIS CTF Finals 2019                        113.624
  8 HITCON CTF 2019 Quals                        89.630
  3 Insomni'hack 2019                            84.343
 14 TokyoWesterns CTF 5th 2019                   83.467
  7 0CTF/TCTF 2019 Finals                        76.567
    Teaser Dragon CTF 2019                       72.300
 10 PlaidCTF 2019                                70.097
```
Which totals to 1092.653 (due to rounding issues, the actual score is 1092.652)

Note that balsn would only get 50 weight max due to first time, but since their lowest top 10 score is 60, they won't get any boost from hosting event.

Hope this helps!

**Comment by KORraNpl** [comment]
Got it, now it makes sense! Thanks for explanation.
